modules/javafx.graphics/src/main/native-glass/win/GlassClipboard.cpp

```diff
@@ -403,6 +403,13 @@ HRESULT PopMemory(
                 //as well as corrupted format
                 cdata = 0;
             }
+        } else if(CF_TEXT == cf || CF_UNICODETEXT == cf){
+            for(int i = 0; i < cdata - 1; i+=2){
+                jbyte *pos = me.getMem() + offset + i;
+                if(*(pos) == 0 && *(pos + 1) == 0){
```

Comment:
1. `offset` is always 0 here.
2. Minor: add spacing around operators and after keywords (`for`, `if`).